pages/listings/listing/edit/index.html

```diff
@@ -33,15 +33,15 @@ <h1 class="text-center mt-4">Edit Listing</h1>
               </div>
               <div class="form mb-3">
                 <label for="location">Location</label>
-                <input class="form-control bg-theme-light" style="border-radius: 0px" type="text" name="location" id="createLocation" required />
+                <input class="form-control bg-theme-light" style="border-radius: 0px" type="text" name="location" id="createLocation" required  aria-label="Put in location" />
```

Comment:
aria-label should be "Enter Location" to ensure consistency across documents, now that the create listings new listings form uses it

Reply:
Fix that also now